src/helpers/helperUtils.js

```diff
@@ -0,0 +1,57 @@
+import dotenv from 'dotenv';
```

Comment:
Why did you delete ` src/controllers/emailController.js` and `src/helpers/default.js` files?